app/config.py

```diff
@@ -63,12 +63,19 @@ def __getitem__(self, key):
 
         val = self
         for key in keys:
-            val = val.get(key)
-            if val is None:
-                return None
+            if isinstance(val, dict):
+                val = dict.__getitem__(val, key)
+            else:
+                raise KeyError(key)
 
         return val
 
+    def get(self, key, default=None, /):
```

Comment:
I don't understand the need for a slash at the end here.

Reply:
It is the same signature as what Python has for its dictionary; it means you cannot specify any arguments before the slash as keyword arguments.  I.e., you cannot write `get(key, default='foo')` but have to write `get(key, 'foo')`.  I'm just following the existing Python pattern here (personally, I don't really see the point).

Reply:
Thanks for that explanation